inference-engine/src/mkldnn_plugin/nodes/mkldnn_fullyconnected_node.h

```diff
@@ -29,6 +29,8 @@ class MKLDNNFullyConnectedNode : public MKLDNNNode {
     const std::vector<impl_desc_type>& getPrimitivesPriority() override;
     void createDescriptor(const std::vector<const MemoryDesc*>& inputDesc,
                           const std::vector<const MemoryDesc*>& outputDesc) override;
+    void specificCreateDescriptor(const MemoryDesc* inputDesc,
+                                  const mkldnn::memory::desc &outputDesc);
```

Comment:
Please move to the `private` section and rename to something like `createDescriptorInternal(const mkldnn::memory::desc &inputDesc, const mkldnn::memory::desc &outputDesc)`.